portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  </head>
+  <body>
+
+      <h1 class="jumbotron"> <center> Gallery </center> </h1>
+<nav>
+      <ul>
+   <li><a href="index.html">Home</a></li>
+  <li><a href="about.html">About</a></li>
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+      </ul>
+</nav>
+      
+      
+          
+       <div class="container">
+      <img src="media/portfolioimage.jpg" alt="Eric on a bridge." class="img-circle" style="width:410px;height:410px;">
```

Comment:
I don't see any of these image files in your pull request, you'll need to "git add" them as well